eng/Versions.props

```diff
@@ -83,8 +83,7 @@
   <PropertyGroup Label="Manual">
     <!-- dotnet/runtime packages -->
     <MicrosoftExtensionsDependencyModelPackageVersion>6.0.0</MicrosoftExtensionsDependencyModelPackageVersion>
-    <MicrosoftExtensionsConfigurationPackageVersion>6.0.0</MicrosoftExtensionsConfigurationPackageVersion>
-    <MicrosoftExtensionsLoggingPackageVersion>6.0.0</MicrosoftExtensionsLoggingPackageVersion>
+    <MicrosoftExtensionsPackageVersion>6.0.0</MicrosoftExtensionsPackageVersion>
```

Comment:
I feel like we either need to give this a more detailed name or `MicrosoftExtensionsDependencyModelPackageVersion` should derive from it.

Reply:
Ya I wasn't entirely sure what the MicrosoftExtensionsDependencyModelPackageVersion was meant for to be honest. It's possible we can do away with it entirely.

Reply:
Was able to remove in latest commit. Was carry over from Razor compiler + SDK days